pkg/server/endpoints/authorized_entryfetcher_registration_entries.go

```diff
@@ -40,9 +40,6 @@ type registrationEntries struct {
 }
 
 func (a *registrationEntries) captureChangedEntries(ctx context.Context) error {
-	// first, reset the entries we might fetch.
-	a.fetchEntries = make(map[string]struct{})
```

Comment:
Where is this map being allocated now?

Reply:
It's allocated in `buildRegistrationEntriesCache`, which is how the structure is created.

Reply:
Thanks.